packages/account/src/Components/poa/status/status-codes/status-codes.ts

```diff
@@ -0,0 +1,12 @@
+type T_poa_status_codes = Readonly<
```

Comment:
Shouldn't be like this? `TPoaStatusCodes`

Reply:
Actually I though about it.. but it is not a function. But I like it more, thanks 👍